maro/simulator/scenarios/data_center/cpu_reader.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from maro.data_lib.binary_reader import BinaryReader
+
+
+class CpuReader:
+
+    def __init__(self, data_path: str):
+        self._data_path = data_path
+        self._cpu_reader = BinaryReader(self._data_path)
+        self._cpu_item_picker = self._cpu_reader.items_tick_picker(
+            start_time_offset=self._cpu_reader.header.starttime,
+            end_time_offset=self._cpu_reader.header.endtime,
+            time_unit="s"
+        )
+        self.count = 0
+
+    def _modify_file_name(self, data_path) -> str:
```

Comment:
rename to ``switch_to_next_file_name`` or ``get_next_file_name`` or some clearer naming